src/components/select-field/__tests__/BaseSelectField.test.js

```diff
@@ -127,7 +135,47 @@ describe('components/select-field/BaseSelectField', () => {
         });
     });
 
+    describe('renderSearchInput', () => {
+        test('should render SearchForm if shouldShowSearchInput is true', () => {
+            const wrapper = shallowRenderSelectField({
+                shouldShowSearchInput: true,
+            });
+
+            const searchForm = wrapper.find('SearchForm');
+
+            expect(searchForm.length).toBe(1);
+        });
+    });
+
     describe('renderSelectOptions()', () => {
+        test('should render FormattedMessage if searchText is not a substring of any of the options', () => {
+            const searchText = 'abc';
+            const wrapper = shallowRenderSelectField();
+            const instance = wrapper.instance();
+            instance.setState({
+                searchText,
+            });
+
+            const message = wrapper.find(FormattedMessage);
```

Comment:
Normally I would expect to see this as a string, but I am guessing using the instance works?

Reply:
https://enzymejs.github.io/enzyme/docs/api/ShallowWrapper/find.html

In the docs it looks like we are able to find the component by passing in a component constructor.